### PR TITLE
248 backup imp

### PR DESCRIPTION
**Pull request by lapinlapin**
#### What's this PR do?
-Made new forms
-Added logic for change forms
-Some refactoring of stepConfig
-Some mini fixes
-Editing unlock form
-Fix bg
#### Where should the reviewer start?
views/Backup/BackupMenu
views/Backup/BackupKeyDownload
#### How should this be manually tested?
https://localhost:8080
#### What are the relevant tickets?
#248 
#### Screenshots (if appropriate)
image

#### URL
Provide a link to your deploy from http://staging-ui.trusty.apasia.tech:8080/branch-248-backup-imp

### CODE REVIEW

**Review by roma219 — CHANGES_REQUESTED** [review]
Modal component has been improved for better display (100% width and height on mobile) and now Backup needs massive layout remakes:

1) See `@/views/Deposit/DepositWidget` for example of how such modals should be composed. Width can wary (especially on mobile) so there should not be any hard-coded width or stuff like `width: 80%`
2) Make it adjust to 100% on mobile correctly (see Deposit for example)
3) Remove media-queries
4) cleanup css from unnecessary stuff (`height: auto` etc)

**Comment by lapinlapin** [comment]
@roma219 updated

File: src/store/modules/backup.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const types = {
 }
 
 const state = {
-  showBackupModal: false
+  showBackupModal: true
```
Comment (roma219) [inline src/store/modules/backup.js]: revert

File: src/views/Backup/Backup.vue [hunk 2]
```diff
@@ -1,38 +1,64 @@
...
-  <div class="backup">
+  <div class="backup-container h-full sm:w-120">
     <svgicon
-      v-if="currentStep > 1"
+      v-if="currentStep > 0"
       class="backup-paginator"
       name="arrowDown"
       @click="goBack"
     />
+    <!--<div
```
Comment (roma219) [inline src/views/Backup/Backup.vue]: remove commented

**Review by roma219 — APPROVED** [review]
👍

File: src/store/modules/backup.js (hunk 1 above)
Comment (lapinlapin) [reply src/store/modules/backup.js]: done

**Review by roma219 — APPROVED** [review]

**Pull request merged**